falcon/routing/static.py

```diff
@@ -49,10 +56,13 @@ def __init__(self, prefix, directory, downloadable=False):
         self._prefix = prefix
         self._directory = directory
         self._downloadable = downloadable
+        self._fallback_filename = None if fallback_filename is None else fallback_path
```

Comment:
could this be paired down to `self._fallback_filename = fallback_filename or fallback_path`?

Reply:
No, because we want to assign `fallback_path` when `fallback_filename` is specified by the user